### DISCUSSION TITLE
How to keep more than one folders when use it?

### DISCUSSION THREAD

**Discussion by atline** [newren/git-filter-repo#594] [Q&A] [answered]
I mean I can use next to keep 1 folder and its history:

```
git filter-repo --path order/
```

But, if I want to keep two folders, how I can manage that?
Looks `git filter-repo --path order1/ order2/` can't work.

Thanks!

**Comment by newren** [comment, marked as answer]
Use two --path arguments:
`git filter-repo --path order1/ --path order2/`

just as covered by the very first example in the manual.
